robottelo/utils/issue_handlers/README.md

```diff
@@ -1,87 +1,139 @@
-# Issue tracker handlers
+# Jira Issue Handler
 
-Issue handlers are modules exposing functions that collects and calculates statuses for issue trackers such as Bugzilla, Jira and Github.
+The Jira issue handler (`jira.py`) is currently the only implemented handler and provides comprehensive functionality for managing Jira issues.
 
-## Implementation of issue handler
+### Core Functions
```

Comment:
H2 must have slipped away. 😏 Please adjust the level of the headings in this doc.